app/views/homepage/_links_and_search.html.erb

```diff
@@ -1,42 +1,45 @@
-<section class="homepage-links-and-search">
-  <div class="govuk-width-container">
-    <div class="govuk-grid-row">
-      <div class="govuk-grid-column-one-half govuk-grid-column-two-thirds-from-desktop">
-        <%= render "govuk_publishing_components/components/heading", {
-          font_size: "m",
-          margin_bottom: 4,
-          text: t("homepage.index.most_viewed"),
-        } %>
-        <ul class="homepage-most-viewed-list">
-          <% t("homepage.index.most_viewed_list").each do | item | %>
-            <li>
-              <%= link_to(item[:text], item[:href], {
-                class: "govuk-link govuk-link--no-visited-state homepage-most-viewed-list__item",
-                data: {
-                  track_category: "homepageClicked",
-                  track_action: "popularLink",
-                  track_label: item[:href],
-                  track_value: 1,
-                  track_dimension_index: 29,
-                  track_dimension: item[:text],
-                }
-              }) %>
-            </li>
-          <% end %>
-        </ul>
-      </div>
-      <div class="govuk-grid-column-one-half govuk-grid-column-one-third-from-desktop homepage-search">
-        <%= render "govuk_publishing_components/components/search", {
-          button_text: t("homepage.index.search_button"),
-          inline_label: false,
-          label_margin_bottom: 4,
-          label_size: "m",
-          label_text: t("homepage.index.search_label"),
-          margin_bottom: 0,
-          wrap_label_in_a_heading: true,
-          size: "large",
-        } %>
+<div class="homepage-links-and-search">
```

Comment:
Does this div need to be here as we've removed the class?

Reply:
😢 Sadly it does need to be here as it's needed to set the background colour for that full width section -

```
.homepage-links-and-search {
    background: #f3f2f1;
}
```
This is where, I think, it would be really nice to have a utility style background style we could use. Something like `govuk-!-background-light-grey`. I think I might add that as an issue for `govuk-frontend`

Reply:
Oooooof I just spotted this. You  can swap this out for `govuk-colour("light-grey")` for robustness. It's the same hex!

Reply:
On your original point, is there no way to apply this to the `section` below and reduce nesting?

Reply:
Ah of course, sorry completely missed that. Removed now.